src/strategies/defiplaza/index.ts

```diff
@@ -1,5 +1,6 @@
 import { formatUnits } from '@ethersproject/units';
-import { Multicaller } from '../../utils';
+import { Multicaller, customFetch } from '../../utils';
+
 
 export const author = 'trebel-defiplaza';
```

Comment:
cc @trebel-defiplaza 
```suggestion
export const author = 'timanrebel';
```

Reply:
Good reminder to merge these two accounts. Confirmed.